spec/lib/fields/date_spec.rb

```diff
@@ -0,0 +1,6 @@
+require "spec_helper"
+require "administrate/field/date"
+require "support/field_matchers"
+
+describe Administrate::Field::Date do
```

Comment:
These are touched on in a previous spec, but I feel uncomfortable having these empty. Would you be able to add something for this and the `time_spec.rb` file?